core/contextmenu.ts

```diff
@@ -124,11 +124,17 @@ function populate_(
     menu.addChild(menuItem);
     menuItem.setEnabled(option.enabled);
     if (option.enabled) {
-      const actionHandler = function(this: ContextMenuOption) {
+      const actionHandler = function() {
         hide();
-        this.callback(this.scope);
+        // TODO: Change this to create two separate actionHandlers, one for
+        //   the modern ContextMenuOption and one for the legacy.
+        // If .scope does not exist on the option, then the callback will not
+        // be expecting a scope parameter, so there should be no problems. Just
+        // assume it is a ContextMenuOption and we'll pass undefined if it's
+        // not.
+        option.callback((option as ContextMenuOption).scope);
       };
-      menuItem.onAction(actionHandler, option);
+      menuItem.onAction(actionHandler, {});
```

Comment:
Yeah this does look weird, but it's correct. `option` would just bind the `option` object to `this`. But I changed it to no longer use `this` to make closure happy. So I also removed passing `option` to `onAction` so that it was clear we weren't using `this`.